client/i18n/locales/english/translations.json

```
@@ -495,7 +495,7 @@
     "progress-modal-cta-8": "Donate now to help us develop new courses on emerging tools and programming concepts.",
     "progress-modal-cta-9": "Donate now to support our math for developers curriculum.",
     "progress-modal-cta-10": "Donate now to help us develop free professional programming certifications for all.",
-    "help-us-reach-goal": "Donate now to help our charity reach our goal of 10,000 monthly supporters by 2024.",
+    "help-us-reach-goal": "Donate now to help our charity reach its goal of 10,000 monthly supporters by 2024.",
```

```suggestion
    "help-us-reach-goal": "Donate now to help our charity reach 10,000 monthly supporters by 2024.",
```

Maybe shortening it? 🤔